org.palladiosimulator.analyzer.slingshot.behavior.resourcesimulation/src/org/palladiosimulator/analyzer/slingshot/behavior/resourcesimulation/resources/active/AbstractActiveResource.java

```diff
@@ -65,6 +59,23 @@ public AbstractActiveResource(final ActiveResourceCompoundKey id, final String n
 	 */
 	protected abstract Optional<AbstractJobEvent> process(final JobInitiated jobInitiated);
 
+	/**
+	 * The delegated handler of the resource that will be processed if the job
+	 * belongs to this resource according to {@link #jobBelongsToResource(Job)}
+	 * implementation.
+	 *
+	 * @param jobProgressed The event.
+	 * @return The appropriate events.
+	 */
+	protected abstract Set<AbstractJobEvent> process(final JobProgressed jobProgressed);
+
+	/**
+	 * TODO
```

Comment:
?

Reply:
ups :x guess i forgot about that one. going to fix it soon.